sdv/multi_table/base.py

```diff
@@ -56,13 +59,14 @@ def _set_temp_numpy_seed(self):
             np.random.set_state(initial_state)
 
     def _initialize_models(self):
-        for table_name, table_metadata in self.metadata.tables.items():
-            synthesizer_parameters = self._table_parameters.get(table_name, {})
-            self._table_synthesizers[table_name] = self._synthesizer(
-                metadata=table_metadata,
-                locales=self.locales,
-                **synthesizer_parameters
-            )
+        with disable_single_table_logger():
```

Comment:
I'm wondering if it would be more straightforward to have the single table synthesizer log conditionally ie.
```python
if synthesizer.is_part_of_multitable: # we can have a better name
    SYNTHESIZER_LOGGER.info(...)
```
To do this we'd need to have some attribute that tracks if the synthesizer was made as part of a multitable synthesizer, but I think that could be useful information. @frances-h @pvk-developer What do you think?

Reply:
I agree. I can add the `table_name` for the `synthesizer` and add it to the `data_processor` as well. Then the logic would be something like 
```python
if not synthesizer.table_name:
    # log the message